experimenter/experimenter/nimbus-ui/src/components/Summary/TableBranches/index.test.tsx

```diff
@@ -177,6 +180,21 @@ describe("TableBranches", () => {
     expect(branchTables).toHaveLength(3);
 
     const subjectTable = branchTables[1];
+
+    const screenshotsAnchorId = `branch-${encodeURIComponent(
+      branchSlug,
+    )}-screenshots`;
+    const screenshotsAnchor = subjectTable.querySelector(
+      "[data-testid='branch-screenshots-anchor']",
+    );
+
+    expect(screenshotsAnchor).not.toBeNull();
+    expect(screenshotsAnchor).toHaveAttribute("id", screenshotsAnchorId);
+    expect(screenshotsAnchor).toHaveAttribute(
+      "href",
+      `#${screenshotsAnchorId}`,
+    );
```

Comment:
We should have warnings for this but we don't. Please use react testing library for this.

```suggestion
    const screenshotsAnchors = screen.getAllByTestId("branch-screenshots-anchor");
    
    expect(screenshotsAnchors).toHaveLength(3);
    expect(screenshotsAnchors[0]).toHaveAttribute("id", screenshotsAnchorId)
    expect(screenshotsAnchors[0]).toHaveAttribute(
      "href",
      `#${screenshotsAnchorId}`,
    );
```

getAllByTestId will throw if there are no results, so we don't need to assert on its return.